src/storage/file_manager.h

```diff
@@ -215,6 +215,8 @@ extern DISK_ISVALID file_tracker_check (THREAD_ENTRY * thread_p);
 extern int file_tracker_dump (THREAD_ENTRY * thread_p, FILE * fp);
 extern int file_tracker_dump_all_capacities (THREAD_ENTRY * thread_p, FILE * fp);
 extern int file_tracker_dump_all_heap (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records);
+extern void file_class_dump_specific_heap_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records,
```

Comment:
Based on the determined processing method, this function should be defined in heap_file.c/h. And this function might call the following functions to convert class name into hfid.
- xlocator_find_class_oid()
- heap_hfid_cache_get()

Reply:
I moved the function to heap_file.c and change its name to `heap_dump_specific_file`.